lib/github.js

```diff
@@ -0,0 +1,14 @@
+const fs = require('fs');
+
+function readGithubToken() {
+  const credentials = fs
+    .readFileSync('~/.git-credentials')
```

Comment:
Please use a async method and the same one of the core module (readFile) but with await. As you can see I'm not using anything but async/await for asynchronous stuff everywhere.

Reply:
To get the global path of the use is better to use this (https://www.npmjs.com/package/home-or-tmp), because the option you chose doesn't work on Windows.

Reply:
Thank you, I've used `os.homedir()` - seems like it works properly here :)

Reply:
I'm not sure about `await` - [fs.readFile](https://nodejs.org/api/fs.html#fs_fs_readfile_path_options_callback) doesn't return promise, but use  callback. Is it possible to use `async/await` with callbacks ?

Reply:
No, it is not. But they also provide promises since some versions ago: https://nodejs.org/api/fs.html#fs_fspromises_readfile_path_options